### PR TITLE
Fix installation of CSVkit and add maintainer

### PR DESCRIPTION
**Pull request by Rub21**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,6 @@
 FROM ubuntu:16.04
+LABEL maintainer "devellopmentseed"
```
Comment (geohacker) [inline Dockerfile]: @Rub21 typo.

File: README.MD [hunk 2]
```diff
@@ -67,7 +67,7 @@ geokit bbox2fc --bbox="0, 0, 10, 10" > output.geojson
 
 ### Buffer
 
-- Creates a buffer in LineString features for a given radius.
+- Creates features' buffer for a given radius.
```
Comment (geohacker) [inline README.MD:70]: 'Create a buffer of specified radius for all the features'

File: README.MD [hunk 3]
```diff
@@ -212,9 +212,9 @@ From: https://github.com/mapbox/tile-cover
 
 ```
 
-### Polygon to point 
+### Feature to point 
 
-- Gets the centroide of polygons an multipolygons.
+- Gets the centroide from features.
```
Comment (geohacker) [inline README.MD]: `Centroid`

File: README.MD (hunk 2 above)
Comment (Rub21) [reply README.MD:70]: yes, it is for all the features.

**Comment by Rub21** [comment]
Thank you @geohacker !!, i will merge this branch

**Pull request merged**